sdk/containerregistry/container-registry/swagger/README.md

```diff
@@ -87,3 +87,113 @@ directive:
     transform: >
       $.required = true
 ```
+
+### Take stream as manifest body
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.parameters.ManifestBody
+  transform: >
+    $.schema = {
+        "type": "string",
+        "format": "binary"
+      }
+```
+
+# Add content-type parameter
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.paths["/v2/{name}/manifests/{reference}"].put
+  transform: >
+    $.parameters.push({
+        "name": "Content-Type",
+        "in": "header",
+        "type": "string",
+        "description": "The manifest's Content-Type."
+    });
+    delete $.responses["201"].schema;
+```
+
+# Change NextLink client name to nextLink
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.parameters.NextLink
+  transform: >
+    $["x-ms-client-name"] = "nextLink"
```

Comment:
hmm, interesting. the code gen doesn't convert this to proper JS casing?

Reply:
or is this .NET thing?

Reply:
Yeah these are just copied from .NET, we can remove them.